### PR TITLE
Add basic structure 2

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no FAQ section. #16 

**What is the new behavior (if this is a feature change)?**
- The basic structure of the FAQ section has been added. #16

### CODE REVIEW

File: src/components/organisms/Sections/FAQ/index.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const items = [
+    {
+        icon: "leaf",
+        heading: "Wie brät man Lorem Ipsum?",
+        text: "Lorem Ipsum dolor sit amet",
+        link: ""
+    },
+    {
+        icon: "leaf",
+        heading: "Schmeckt dieser Kaffee gleich?",
```
Comment (kleberbaum) [inline src/components/organisms/Sections/FAQ/index.jsx]: This doesn't sound like a proper German question. xD

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/Sections/FAQ/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/organisms/Sections/FAQ/index.jsx]: Content is not the objective of this pull request.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**